src/main/ui/src/components/SearchByName/SearchByName.jsx

```diff
@@ -29,38 +34,51 @@ export default function SearchByName() {
 
 
     function filterParkNames(park) {
-        if (searchTerm === ""){
-            return park
-        } if (park.fullName.toLowerCase().includes(searchTerm.toLowerCase())) {
-            return park
-       }
-    }
+            if (searchTerm === ""){
+                return park
+            } if (park.fullName.toLowerCase().includes(searchTerm.toLowerCase())) {
+                return  park
+           }
+        }
+
+    return (
+        <div>
+          <input
+            type="text"
+            placeholder="Search parks by name"
+            onChange={handleSearch}
+          />
 
-return (
+          {data
+            .filter((park) => {
+              return filterParkNames(park);
+            })
+            .map((park, key) => {
+              return (
+                <div key={key}>
+                  <Row xs={2} md={2} className="g-4" >
 
-     <div>
-             <input
-                 type="text"
-                 placeholder="Search parks by name"
-                 onChange = {handleSearch}
-             />
+                       {Array.from({ length: 2 }).map((_, idx) => (
```

Comment:
This is what is making the duplicates. Change to `length: 1` and adjust so each row has two items in it. RIght now it looks like these are arranged with two columns instead of one col and two items per row.

Reply:
Emily told me she can resolve that I will talk to her today